src/components/forms/ProxyDeviceForm.tsx

```diff
@@ -0,0 +1,369 @@
+      getConfigurationRowBase({
+        className: "no-border-top custom-device-name",
+        configuration: (
+          <RenameDeviceInput
+            name={device.name}
+            index={index}
+            setName={(name) => {
+              ensureEditMode(formik);
+              void formik.setFieldValue(`devices.${index}.name`, name);
+            }}
+          />
+        ),
+        inherited: "",
+        override: (
+          <Button
+            className="u-no-margin--top u-no-margin--bottom"
+            onClick={() => {
+              ensureEditMode(formik);
+              removeDevice(index, formik);
+            }}
+            type="button"
+            appearance="base"
+            hasIcon
+            dense
+            title="Detach Proxy"
+          >
+            <Icon name="disconnect" />
+            <span>Detach</span>
+          </Button>
+        ),
+      }),
+    );
+    customRows.push(
+      getProxyDeviceFormRows("NAT mode", "nat", index, [
+        { label: "Enabled", value: "true" },
+        { label: "Disabled", value: "false" },
+      ]),
+    );
+    customRows.push(
+      getProxyDeviceFormRows("Bind", "bind", index, [
+        { label: "Host", value: "host" },
+        { label: "Instance", value: "instance" },
+      ]),
+    );
+    customRows.push(
+      getProxyDeviceFormRows("Use HAProxy Protocol", "proxy_protocol", index, [
+        { label: "Yes", value: "true" },
+        { label: "No", value: "false" },
+      ]),
+    );
+
+    customRows.push(
+      getConfigurationRowBase({
+        className: "no-border-top inherited-with-form",
+        configuration: <Label>Listen</Label>,
+        inherited: (
+          <>
+            <Input
+              name={`devices.${index}.listen`}
+              id={`devices.${index}.listen`}
+              key={`devices.${index}.listen`}
+              onBlur={formik.handleBlur}
+              onChange={(e) => {
+                void formik.setFieldValue(
+                  `devices.${index}.listen`,
+                  e.target.value,
+                );
+              }}
+              // value={formik.values.devices[index][fieldName]}
+              value={""}
+              type="text"
+              className="u-no-margin--bottom"
+              placeholder="0644"
+              help={
+                <ConfigFieldDescription
+                  description={"Mode for the listening Unix socket"}
+                />
+              }
+              label="Mode"
+            />
+            <Input
+              name={`devices.${index}.listen`}
+              id={`devices.${index}.listen`}
+              key={`devices.${index}.listen`}
+              onBlur={formik.handleBlur}
+              onChange={(e) => {
+                void formik.setFieldValue(
+                  `devices.${index}.listen`,
+                  e.target.value,
+                );
+              }}
+              // value={formik.values.devices[index][fieldName]}
```

Comment:
```suggestion
value={device.listen}
```
This should work and also satisfy the TS linter.